go.mod

```diff
@@ -143,16 +143,16 @@ require (
 	helm.sh/helm/v3 v3.9.0
 	k8s.io/api v0.25.4
 	k8s.io/apiextensions-apiserver v0.25.4
-	k8s.io/apimachinery v0.25.4
-	k8s.io/apiserver v0.25.4
+	k8s.io/apimachinery v0.27.2
+	k8s.io/apiserver v0.27.2
 	k8s.io/cli-runtime v0.25.4
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/gengo v0.0.0-20220902162205-c0856e24416d
 	k8s.io/helm v2.16.7+incompatible
 	k8s.io/kube-aggregator v0.25.4
 	k8s.io/kubectl v0.25.4
 	k8s.io/kubernetes v1.25.4
```

Comment:
Any issues on bumping specific `k8s.io` versions and not others? Not sure what is the upstream support on that front, but I would assume that `k8s.io/apiextensions-apiserver` and `k8s.io/api` should always be on the same minor of `k8s.io/apiserver`.

Reply:
I have reverted this change